### PR TITLE
Your Idea worked.

### PR DESCRIPTION
**Pull request by hughbrien**
With respect to the Instrumentation Testing 

I am not sure how to "test" the "async" code, because there needs to "another thread" 

The idea of for including Tornado worked great.

### CODE REVIEW

**Comment by pglombardo** [comment]
Hey @hughbrien - thanks for the PR!  To start to merge this stuff though, let's break this into 3 separate PRs and then we can work, test & validate each one individually.

A couple points:
* Delete vcs.xml file - it's not needed for this repository
* Delete this PR and make 1 each for aiomysql, aiohttp & tornado
* Add tests for each

File: instana/instrumentation/instrument_aiohttp.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+from __future__ import absolute_import
+import opentracing.ext.tags as ext
+import instana
+import opentracing
+import wrapt
+
+
+
+try:
+    import instrument_aiohttp
```
Comment (pglombardo) [inline instana/instrumentation/instrument_aiohttp.py]: This should be attempting to import the aiohttp package.  Here you might have mistakenly put the module name itself (instrument_aiohttp).  If aiohttp is available then we instrument, if not then pass.

File: instana/instrumentation/instrument_aiomysql.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+import instana
+import wrapt
+import opentracing.ext.tags as ext
+
+try:
+    import instrument_aiomysql
```
Comment (pglombardo) [inline instana/instrumentation/instrument_aiomysql.py]: Same comment as aiohttp.

File: instana/instrumentation/instrument_tornado.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+from __future__ import print_function
+
+import opentracing.ext.tags as ext
+import opentracing as ot
+import instana
+import opentracing
+import wrapt
+
+try:
+	import instrument_tornado
```
Comment (pglombardo) [inline instana/instrumentation/instrument_tornado.py]: Same comment as aiohttp.

**Comment by hughbrien** [comment]
Good idea. I will re-fork and do it in three parts.

**Pull request closed without merging**